payjoin-ffi/python/test/test_payjoin_integration_test.py

```diff
@@ -78,56 +78,56 @@ async def process_receiver_proposal(self, receiver: ReceiveSession, recv_persist
             return await self.process_provisional_proposal(receiver.inner, recv_persister)
         if receiver.is_PAYJOIN_PROPOSAL():
             return receiver
-        
+
         raise Exception(f"Unknown receiver state: {receiver}")
-            
-            
+
+
```

Comment:
Were these whitespace changes included in prior reviews of this PR? They seem unrelated. I might just remove them before merging to move this along